gitbook-docs/market/front-end.md

```diff
@@ -1,137 +1,137 @@
 ---
-description: Proof Market front end guide
+description: The web interface of the Proof Market
 ---
 
-# Front End
+# Web interface
 
 {% hint style="info" %}
 Please note this product is in beta and expected to change.
 {% endhint %}
 
-#### Proof Market front end can be accessed at [**proof.market**](https://proof.market)
+#### The web interface of the Proof Market can be accessed at [**proof.market**](https://proof.market).
```

Comment:
It's not a good text for a heading: it's a complete sentence with a link. So it should be just a paragraph.

Reply:
fixed